package.json

```
@@ -24,18 +23,17 @@
     "css-loader": "^6.7.1",
     "css-minimizer-webpack-plugin": "^4.1.0",
     "element-closest": "^3.0.2",
-    "feather-icons": "^4.29.0",
     "file-loader": "^6.2.0",
     "flagpack-dart-sass": "^1.3.0",
     "history": "^5.3.0",
     "htmldiff-js": "1.0.5",
+    "lucide-preact": "^0.88.0",
```

```suggestion
    "lucide": "^0.91.0",
    "lucide-preact": "^0.88.0",
```
> The main difference between devDependencies and "normal" dependencies are the usage in the bundled frontend version. For example: We are using some minifiers and converters like babel that are required for the bundling of the resulting JS file. As lucide is a pure frontend library that is required for our frontend operations I would put it as a "normal" dependency

I would argue that all of our dependencies are dev dependencies, since we don't use npm in production.
Normally, the separation is used to execute `npm install --production` on your production system which would then only install the dependencies without the dev dependencies. In our case however, the prod server does not have npm installed and doesn't know our `package.json` and just gets the bundled static files.